float64 d_yaw     # delta yaw angle in degree
float64 d_pitch   # delta pitch angle in degree
float64 d_roll    # delta roll angle in degree